src/web/errorHandler.ts

```diff
@@ -46,11 +46,18 @@ const handleDefault = function handleDefault(
   res: Response,
   next: NextFunction
 ): void {
+  logger.error('Unhandled error caught by middleware stack')
+  logger.error(error.stack)
+
   if (res.headersSent) {
     return next(error)
   }
+  if (!req.isAuthenticated()) {
+    // don't render application structure for non authenticated issues
+    return res.redirect('/auth/unauthorised?default_throw')
```

Comment:
is the query parameter being used somehow or is it just for indication in url?

Reply:
Just an indication, it should be logged before this point too, this should never really be hit so I'll leave it to differentiate.

Reply:
👍